doc/userguide/rules/dns-keywords.rst

```diff
@@ -46,6 +48,14 @@ Match on DNS requests where the **opcode** is NOT 0::
 
   dns.opcode:!0;
 
+Match on DNS requests where the **opcode** is between 7 and 15, exclusively:
+
+  dns.opcode:7-15;
```

Comment:
Is the range [7-15](inclusive) or [7-15) (exclusive)?

Reply:
Probably a good idea to add a link to https://github.com/OISF/suricata/pull/10197/files#diff-e2d2026f582d7be9e0b729676d5c253ce8a319c64ad6dae5bc38ed1b62377f00R32-R33 section if these are related.

Reply:
It is exclusive. Putting it more explicitly in the doc